spec.md

```diff
@@ -396,6 +396,12 @@ message CreateVolumeRequest {
   // This field is OPTIONAL. The Plugin is responsible for parsing and
   // validating these parameters. COs will treat these as opaque.
   map<string, string> parameters = 5;
+
+  // OPTIONAL field for storing principal information to be used in
+  // the creation of this volume. If the principal is present the
+  // plugin MAY ensure the volume created will be accessible for the
+  // herein specified entities only.
+  Principal volume_root_owner = 6;
```

Comment:
It appears that this field `volume_root_owner` may be different than the `container_acts_as` field. How is a CO supposed to reason about the relationship between these fields? `Principal` is not a trivial structure and it's unclear to me how a CO is supposed to understand, for example, relationship between namespaces, users, and groups (which, on some systems may exist within some hierarchical structure).